app/classes/query/modules/lookup_names.rb

```diff
@@ -80,8 +80,10 @@ def find_exact_name_matches(vals)
           if /^\d+$/.match?(val.to_s)
             result << minimal_name_data(Name.safe_find(val))
           else
-            # Cop appears to generate false positive
+            # rubocop:disable Lint/UselessAssignment
+            # cop generates a false positive
             result += find_matching_names(val)
```

Comment:
I don't think this is a false positive, but is a bit subtle. += does two things.  In this case only the '+' is used not the '=' bit.  I believe you will get the exact same result if you change this to `result + find_matching_names(val)`.